src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx

```diff
@@ -13,32 +14,35 @@ import {useTokenInfo} from '../../hooks'
 import globalMessages, {txLabels} from '../../i18n/global-messages'
 import {getDefaultAssetByNetworkId} from '../../legacy/config'
 import {decodeHexAscii, formatTokenAmount, getAssetDenominationOrId, getTokenFingerprint} from '../../legacy/format'
+import {TxHistoryRouteNavigation} from '../../navigation'
 import {useSelectedWallet} from '../../SelectedWallet'
 import {COLORS} from '../../theme'
 import {Token} from '../../types'
 import {YoroiWallet} from '../../yoroi-wallets'
 import {Quantity, TokenId, YoroiAmounts} from '../../yoroi-wallets/types'
 import {Quantities} from '../../yoroi-wallets/utils'
+import {useSend} from '../Context/SendContext'
 
 type Props = {
   balances: YoroiAmounts
-  onSelect: (tokenId: TokenId) => void
-  onSelectAll: () => void
 }
 
-export const AssetSelectorScreen = ({balances, onSelect, onSelectAll}: Props) => {
+export const AssetSelectorScreen = ({balances}: Props) => {
   const strings = useStrings()
   const wallet = useSelectedWallet()
   const defaultAsset = getDefaultAssetByNetworkId(wallet.networkId)
   const [matcher, setMatcher] = React.useState('')
+  const navigation = useNavigation<TxHistoryRouteNavigation>()
+  const {sendActions} = useSend()
+
   const onChangeMatcher = (matcher: string) => {
     LayoutAnimation.configureNext(LayoutAnimation.Presets.easeInEaseOut)
     setMatcher(matcher)
   }
 
   const sortedBalance: Array<[TokenId, Quantity]> = Object.entries(balances)
     .sort(([, quantityA]: [TokenId, Quantity], [, quantityB]: [TokenId, Quantity]) =>
-      Quantities.isGreaterThan(quantityA, quantityB) ? -1 : 1,
+      Quantities.isGreaterThan(quantityA, quantityB) === true ? -1 : 1,
```

Comment:
I think @wolverineks already suggested it, but we will need to parseInt. maybe keep it this way.
```suggestion
      parseInt(Quantities.diff(quantityA, quantityB), 10),
```

Reply:
Done thanks !